build.gradle

```diff
@@ -19,7 +19,7 @@ buildscript {
 apply from: 'intellijJVersions.gradle'
 
 group 'software.aws.toolkits'
-version toolkitVersion
+version toolkitVersion + "-" + resolveIdeProfileName()
```

Comment:
this is going to be "1.8-2019.2" right ? I think it might be better to go for the 3 digit variant - 1.8-193

Thoughts?

Reply:
I didn't have a preference. We just easily had that data in that format 😄